sql/snowflake/SnowflakeParser.g4

```diff
@@ -2308,10 +2356,16 @@ create_table
     ) ((comment_clause? create_table_clause) | (create_table_clause comment_clause?))
     ;
 
+column_decl_item_list_paren
+    : '(' column_decl_item_list ')'
+    ;
+
 create_table_clause
-    : '(' column_decl_item_list ')' cluster_by? stage_file_format? (
+    : ( column_decl_item_list_paren cluster_by?
+      | cluster_by? comment_clause? column_decl_item_list_paren
+      ) stage_file_format? (
         STAGE_COPY_OPTIONS EQ LR_BRACKET copy_options RR_BRACKET
-    )? (DATA_RETENTION_TIME_IN_DAYS EQ num)? (MAX_DATA_EXTENSION_TIME_IN_DAYS EQ num)? change_tracking? default_ddl_collation? copy_grants?
+    )? (DATA_RETENTION_TIME_IN_DAYS EQ num)? (MAX_DATA_EXTENSION_TIME_IN_DAYS EQ num)? change_tracking? default_ddl_collation? copy_grants? comment_clause?
```

Comment:
comment_clause is already in create_table

Reply:
comment clause can be placed in 2 different places, this needs refactoring

Reply:
Hum I see, thank you